contract/models/contract_recurrency_mixin.py

```diff
@@ -48,6 +48,8 @@ class ContractRecurrencyBasicMixin(models.AbstractModel):
         help="Invoice every (Days/Week/Month/Year)",
     )
     date_start = fields.Date()
+    date_end = fields.Date()
+
```

Comment:
Don't add empty lines here in the middle.

Reply:
My bad. Sorry.